src/app/public/modules/tabs/tabset.component.spec.ts

```diff
@@ -227,11 +227,6 @@ describe('Tabset component', () => {
     });
   });
 
-  describe('Xs tab heading', () => {
```

Comment:
Removed any skipped unit tests since these are deprecated (and the test cases are met elsewhere).